### ISSUE TITLE
Duplicate declaration "Miss" in the example

### ISSUE THREAD

**Issue by hezjing** [ReactTraining/react-router#3841]
There is duplicate declaration "Miss" in the [example](https://react-router-website-uxmsaeusnn.now.sh/quick-start):
```
  44 | )
  45 |
> 46 | const Miss = ({ location }) => (
     |       ^
  47 |   <div>
  48 |     <h2>Whoops</h2>
  49 |     <p>Sorry but {location.pathname} didn't match any pages</p>
```

**Comment by davorbadrov** [comment]
If that is from the quick start example, based on the docs I think that should be:

`const NoMatch = ...`

Miss is React Router's intern directive

**Comment by chentsulin** [comment]
Already fixed by 7ef3b0a
https://github.com/ReactTraining/react-router/commit/7ef3b0a84a051659ded829641643e58eba8a4db1#diff-e2c8ad2381ba3aacc17c2c351564d7e4

**Comment by threepointone** [comment]
As specified by @chentsulin, this is already fixed, and the new docs reflect it.
